source/config.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+const os = require('os');
+const isInstalledGlobally = require('is-installed-globally');
+const cosmiconfig = require('cosmiconfig');
+
+module.exports = async () => {
+	const searchDir = isInstalledGlobally ? os.homedir() : process.cwd();
```

Comment:
It should not be `process.cwd()`, but rather the same directory as package.json. They will most likely be the same, but the user could have `cd foo && np` in their run script, in which, this logic would be incorrect.